rotkehlchen/api/v1/schemas.py

```diff
@@ -3065,13 +3065,18 @@ class AccountingRuleIdSchema(Schema):
     counterparty = fields.String(required=False, load_default=None)
 
 
+class LinkedAccountingSetting(Schema):
```

Comment:
Small docstring explaining the schema. What is the value and how is it linked to the linked setting